docs-guides/source/opt-quantization-api.md

```diff
@@ -9,12 +9,12 @@ You can linearly quantize the weights of your Core ML model by using the
 [``linear_quantize_weights``](https://apple.github.io/coremltools/source/coremltools.optimize.coreml.post_training_quantization.html#coremltools.optimize.coreml.linear_quantize_weights) method as follows:
 
 ```python
-import coremltools.optimize.coreml as cto
+import coremltools.optimize as cto
 
-op_config = cto.OpLinearQuantizerConfig(mode="linear_symmetric", weight_threshold=512)
-config = cto.OptimizationConfig(global_config=op_config)
+op_config = cto.coreml.OpLinearQuantizerConfig(mode="linear_symmetric", weight_threshold=512)
```

Comment:
Can you format the code so that `weight_threshold` is on the next line, otherwise with this change , the user would have to scroll the code block

<img width="848" alt="image" src="https://github.com/apple/coremltools/assets/7546478/0ee0426c-b317-456c-be6d-06a03b868063">